### PR TITLE
Task 1

### PR DESCRIPTION
**Pull request by ps99**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+import {mockCourses} from './mockfile'// eslint-disable-next-line
+import {CourseModel} from './mockfileModel'// eslint-disable-next-line
```
Comment (pkhlebko) [inline src/index.tsx:8]: Unused variable?

File: src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
+
+import {mockCourses} from './mockfile'// eslint-disable-next-line
```
Comment (pkhlebko) [inline src/index.tsx:7]: It works OK without `// eslint-disable-next-line here`. Why did you add it?

File: src/components/Course/Course.tsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+export const Course = (props:any) => {
+  return (
+    <li className="course_card__wrapper">
+      <div className="course_card">
+        <div className="course_card__title">
+          {props.data.title}
+          <em className="course_card__date">(Published on {props.data.date})</em>
```
Comment (pkhlebko) [inline src/components/Course/Course.tsx:7]: props.data ... It seems this is a good place to use destructuring assignment following DRY principle.

**Review by pkhlebko — APPROVED** [review]
LGTM. Added some cosmetic comments

File: src/index.tsx (hunk 2 above)
Comment (ps99) [reply src/index.tsx:7]: Good question :)
Seems, I tried to find cause of an issue and forgot to remove this line.

File: src/index.tsx (hunk 1 above)
Comment (ps99) [reply src/index.tsx]: Fixed

File: src/index.tsx (hunk 2 above)
Comment (ps99) [reply src/index.tsx]: Fixed

File: src/components/Course/Course.tsx (hunk 3 above)
Comment (ps99) [reply src/components/Course/Course.tsx]: Updated with destructuring assignment
